src/client/components/FriendsList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useAppDispatch, useAppSelector } from '../state/hooks';
+import { BackArrow, ProfileImage, OrangeButton, OrangeNavbar } from '../styles/shared';
+
+const FriendsListDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Text = styled.span`
+  margin-top: 50px;
+  margin-bottom: 25px;
+`;
+
+const List = styled.ul`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ListItem = styled.li`
+  display: flex;
+`;
+
+const ProfilePic = styled.span`
+  margin-right: 10px;
+`;
+
+const FriendsList = (props) => {
+  const currentUser = useAppSelector((state) => state.currentUser);
+
+  return (
+    <FriendsListDiv>
+      <OrangeNavbar needBackArrow={true} />
```

Comment:
to have the back arrow functioning, dont forget to put the second prop and import the following
import { useHistory } from 'react-router-dom'
< OrangeNavbar needBackArrow={true} onBackArrowClick={useHistory().goBack} />

Reply:
yeah I was planning on adding in the back functionality towards the end since i'm still working on other things